### DISCUSSION TITLE
How can i contribute

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22815] [New to GitHub]
<p>Hi I’m Surya and I have a question how can I contribute in Github</p>
<p>please help</p>
<blockquote>
<p>Blockquote Surya</p>
</blockquote>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hello <a class="mention" href="/suryachandrakumar987">@suryachandrakumar987</a> :wave:</p>
<p>We are happy you are here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right directions.</p>
<p>If you are looking to contribute to a specific repository on GitHub, please check the repository <a href="https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-readmes">Readme</a> file. This is the first place to start when learning about the process of contributing to a specific project.</p>
<p>Please also refer to this great <a href="https://github.community/t/best-practices-for-open-source-software-contributions/10209">Team</a> post about contributing to open-source software.</p>
<aside class="quote quote-modified" data-post="1" data-topic="10209">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/best-practices-for-open-source-software-contributions/10209">Best practices for Open Source Software Contributions</a> <a class="badge-wrapper  bullet" href="/c/github-original-series/support-protips/29"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #BF1E2E;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Written by our Support experts, this series brings you tips and tricks to help you improve your GitHub workflows and experience.">Support Protips</span></a>
  </div>
  <blockquote>
    What’s in it for you and your company? 
“We only have what we give”. I.A. 
You are already here, many of your projects have dependencies buried deep in the OSS ecosystem, and you want to see those open source projects thrive, and more importantly stay alive. Open Source infrastructure is dependent on the work of talented contributors and developers, and those who contribute have a lot to gain for career development and personal fulfillment. 
Companies that invest time and resources to contribute…
  </blockquote>
</aside>
